core/include/gnuradio-4.0/Message.hpp

```diff
@@ -20,11 +20,41 @@ const std::string ErrorInfo = "ERROR_INFO_KEY"; // optional: if a message has an
 
 namespace message::kind {
 const std::string Error           = "ERROR_KIND";
-const std::string Graph_update    = "GRAPH_UPDATE_KIND";
+const std::string GraphUpdate     = "GRAPH_UPDATE_KIND";
+const std::string SchedulerUpdate = "SCHEDULER_UPDATE_KIND";
 const std::string UpdateSettings  = "UPDATE_SETTINGS_KIND";
 const std::string SettingsChanged = "SETTINGS_CHANGED_KIND";
 } // namespace message::kind
 
+namespace message::graph_update {
+namespace what {
+const std::string BlockAdded         = "ADDED_BLOCK_GRAPH_UPDATE";
```

Comment:
I could use some example messages how these would look like in practice. I assume they would also contain the initial block properties?